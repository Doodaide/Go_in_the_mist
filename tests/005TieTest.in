place A1
place B1
place C1
place D1
place E1
place F1
place G1
place H1
place I1
place J1
place K1
place L1
place M1
place N1
place O1
place P1
place Q1
place R1
place S1
place A3
place B3
place C3
place D3
place E3
place F3
place G3
place H3
place I3
place J3
place K3
place L3
place M3
place N3
place O3
place P3
place Q3
place R3
place S3
place A5
place B5
place C5
place D5
place E5
place F5
place G5
place H5
place I5
place J5
place K5
place L5
place M5
place N5
place O5
place P5
place Q5
place R5
place S5
place A7
place B7
place C7
place D7
place E7
place F7
place G7
place H7
place I7
place J7
place K7
place L7
place M7
place N7
place O7
place P7
place Q7
place R7
place S7
place A9
place B9
place C9
place D9
place E9
place F9
place G9
place H9
place I9
place J9
place K9
place L9
place M9
place N9
place O9
place P9
place Q9
place R9
place S9
place A11
place B11
place C11
place D11
place E11
place F11
place G11
place H11
place I11
place J11
place K11
place L11
place M11
place N11
place O11
place P11
place Q11
place R11
place S11
place A13
place B13
place C13
place D13
place E13
place F13
place G13
place H13
place I13
place J13
place K13
place L13
place M13
place N13
place O13
place P13
place Q13
place R13
place S13
place A15
place B15
place C15
place D15
place E15
place F15
place G15
place H15
place I15
place J15
place K15
place L15
place M15
place N15
place O15
place P15
place Q15
place R15
place S15
place A17
place B17
place C17
place D17
place E17
place F17
place G17
place H17
place I17
place J17
place K17
place L17
place M17
place N17
place O17
place P17
place Q17
place R17
place S17
place A19
place B19
place C19
place D19
place E19
place F19
place G19
place H19
place I19
place J19
place K19
place L19
place M19
place N19
place O19
place P19
place Q19
place R19
place S19
place A2
place B2
place C2
place D2
place E2
place F2
place G2
place H2
place I2
place J2
place K2
place L2
place M2
place N2
place O2
place P2
place Q2
place R2
place S2
place A4
place B4
place C4
place D4
place E4
place F4
place G4
place H4
place I4
place J4
place K4
place L4
place M4
place N4
place O4
place P4
place Q4
place R4
place S4
place A6
place B6
place C6
place D6
place E6
place F6
place G6
place H6
place I6
place J6
place K6
place L6
place M6
place N6
place O6
place P6
place Q6
place R6
place S6
place A8
place B8
place C8
place D8
place E8
place F8
place G8
place H8
place I8
place J8
place K8
place L8
place M8
place N8
place O8
place P8
place Q8
place R8
place S8
place A10
place B10
place C10
place D10
place E10
place F10
place G10
place H10
place I10
place J10
place K10
place L10
place M10
place N10
place O10
place P10
place Q10
place R10
place S10
place A12
place B12
place C12
place D12
place E12
place F12
place G12
place H12
place I12
place J12
place K12
place L12
place M12
place N12
place O12
place P12
place Q12
place R12
place S12
place A14
place B14
place C14
place D14
place E14
place F14
place G14
place H14
place I14
place J14
place K14
place L14
place M14
place N14
place O14
place P14
place Q14
place R14
place S14
place A16
place B16
place C16
place D16
place E16
place F16
place G16
place H16
place I16
place J16
place K16
place L16
place M16
place N16
place O16
place P16
place Q16
place R16
place S16
place A18
place B18
place C18
place D18
place E18
place F18
place G18
place H18
place I18
place J18
place K18
place L18
place M18
place N18
place O18
place P18
place Q18
place R18
place S18
