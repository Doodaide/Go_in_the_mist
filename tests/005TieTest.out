Wow, a tie!
A1B1C1D1E1F1G1H1I1J1K1L1M1N1O1P1Q1R1S1A3B3C3D3E3F3G3H3I3J3K3L3M3N3O3P3Q3R3S3A5B5C5D5E5F5G5H5I5J5K5L5M5N5O5P5Q5R5S5A7B7C7D7E7F7G7H7I7J7K7L7M7N7O7P7Q7R7S7A9B9C9D9E9F9G9H9I9J9K9L9M9N9O9P9Q9R9S9A11B11C11D11E11F11G11H11I11J11K11L11M11N11O11P11Q11R11S11A13B13C13D13E13F13G13H13I13J13K13L13M13N13O13P13Q13R13S13A15B15C15D15E15F15G15H15I15J15K15L15M15N15O15P15Q15R15S15A17B17C17D17E17F17G17H17I17J17K17L17M17N17O17P17Q17R17S17A19B19C19D19E19F19G19H19I19J19K19L19M19N19O19P19Q19R19S19A2B2C2D2E2F2G2H2I2J2K2L2M2N2O2P2Q2R2S2A4B4C4D4E4F4G4H4I4J4K4L4M4N4O4P4Q4R4S4A6B6C6D6E6F6G6H6I6J6K6L6M6N6O6P6Q6R6S6A8B8C8D8E8F8G8H8I8J8K8L8M8N8O8P8Q8R8S8A10B10C10D10E10F10G10H10I10J10K10L10M10N10O10P10Q10R10S10A12B12C12D12E12F12G12H12I12J12K12L12M12N12O12P12Q12R12S12A14B14C14D14E14F14G14H14I14J14K14L14M14N14O14P14Q14R14S14A16B16C16D16E16F16G16H16I16J16K16L16M16N16O16P16Q16R16S16A18B18C18D18E18F18G18H18I18J18K18L18M18N18O18P18Q18R18S18
Thank you for playing!
